science/rtl_433/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           cmake 1.1
 PortGroup           github 1.0
 
 epoch               1
-github.setup        merbanan rtl_433 20.11
+github.setup        merbanan rtl_433 21.05
 revision            1
```

Comment:
when updating the version one should reset the `revision` to `0`

Reply:
Oops. Fixed.